src/pages/scientificServices/pipelines/views/TableFilters.tsx

```diff
@@ -0,0 +1,207 @@
+        <StatusFilterControl filters={filters} handleInputChange={handleInputChange} />
+        <PipelineFilterControl filters={filters} handleInputChange={handleInputChange} pipelinesList={pipelinesList} />
+
+        <div
+          style={{
+            display: 'flex',
+            flex: '0 0 auto',
+            justifyContent: 'center',
+          }}
+        >
+          <ButtonPrimary disabled={!hasActiveFilters} onClick={handleClearFilters}>
+            Clear
+          </ButtonPrimary>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const DescriptionFilterControl = ({
+  filters,
+  handleInputChange,
+}: {
+  filters: FilterValues;
+  handleInputChange: (field: keyof FilterValues, value: string) => void;
+}) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.25rem',
+        minWidth: '200px',
+        flex: '1 1 25%',
+      }}
+    >
+      <div style={{ fontWeight: 600, fontSize: '14px' }}>Description</div>
+      <DelayedSearchInput
+        id='filter-description'
+        placeholder='Filter by description...'
+        value={filters.description || ''}
+        onChange={(value: string) => handleInputChange('description', value)}
+      />
+    </div>
+  );
+};
+
+const JobIdFilterControl = ({
+  filters,
+  handleInputChange,
+}: {
+  filters: FilterValues;
+  handleInputChange: (field: keyof FilterValues, value: string) => void;
+}) => {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        flexDirection: 'column',
+        gap: '0.25rem',
+        minWidth: '200px',
+        flex: '1 1 25%',
+      }}
+    >
+      <div style={{ fontWeight: 600, fontSize: '14px' }}>Job ID</div>
+      <DelayedSearchInput
+        id='filter-jobId'
+        placeholder='Filter by Job ID...'
+        value={filters.jobId || ''}
+        onChange={(value: string) => handleInputChange('jobId', value)}
+      />
+    </div>
+  );
+};
+
+const StatusFilterControl = ({
+  filters,
+  handleInputChange,
+}: {
+  filters: FilterValues;
+  handleInputChange: (field: keyof FilterValues, value: string) => void;
+}) => {
+  const statusOptions: { value: PipelineRunStatus; label: string }[] = [
+    { value: 'PREPARING', label: 'Preparing' },
+    { value: 'RUNNING', label: 'In Progress' },
+    { value: 'SUCCEEDED', label: 'Done' },
+    { value: 'FAILED', label: 'Failed' },
+  ];
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', gap: '0.25rem', minWidth: '150px', flex: '1 1 15%' }}>
+      <div style={{ fontWeight: 600, fontSize: '14px' }}>Status</div>
+      <Select
+        id='filter-status'
+        options={statusOptions}
+        value={filters.status}
+        placeholder='All Statuses'
+        styles={{
+          control: (provided) => ({
+            ...provided,
+            minHeight: '2.25rem',
+            height: '2.25rem',
+          }),
```

Comment:
it drove me crazy that the height of the `Select` component was a few pixels taller than the height of the text input components that are next to these, hence the height style override to make them consistent